Jenkinsfile

```diff
@@ -140,6 +140,7 @@ pipeline {
           anyOf {
             branch 'master'
             branch 'release/*'
+            branch 'trying'
```

Comment:
Did you mean to make this change?

Reply:
Yes I did. Should I do a separate PR for it?

Reply:
Ah ok.
Should we skip just the submodule check rather than the entire lint? Reason for asking is that having the lint in place helps make sure code is correctly linted which makes core review easier as they won't be able to test changes on CI if code is not linted properly.

Reply:
I think, that's would be a better solution.